config/rbac/kustomization.yaml

```diff
@@ -12,7 +12,7 @@ resources:
 # Comment the following 4 lines if you want to disable
 # the auth proxy (https://github.com/brancz/kube-rbac-proxy)
 # which protects your /metrics endpoint.
-# - auth_proxy_service.yaml
-# - auth_proxy_role.yaml
-# - auth_proxy_role_binding.yaml
+- auth_proxy_service.yaml
```

Comment:
auth_proxy_service.yaml needs to be updated to reflect the fact that the metrics port's name has changed from "https" to "metrics".

Reply:
@MikeSpreitzer I changed the SVC port `name` and `targetPort`